metricflow-semantics/metricflow_semantics/errors/error_classes.py

```diff
@@ -4,7 +4,13 @@
 from typing import Dict, Optional
 
 
-class CustomerFacingSemanticException(Exception):
+class InformativeError(Exception):
+    """Raised when the error is actionable by the user or is informational."""
```

Comment:
What errors aren't `informational`?

Reply:
Updated to `InformativeUserError` if that's any better and updated the docstring